docs/usage/shoot_scheduling_profiles.md

```diff
@@ -0,0 +1,69 @@
+  
+- `bin-packing` (alpha)
+
+   **Overview**
+
+   The `bin-packing` profile scores Nodes with a high allocation of resources higher than the others, resulting in a greater overall resource utilizaton. By favoring the Nodes with higher allocation some of the other Nodes become underutilized over time (because new Pods keep being scheduled to the most allocated Nodes). Then, the cluster autoscaler identifies such underutilized Nodes and removes them from the cluster. In this way this profile provides a greater overall resource utilization (compared to the `balanced` profile).
+
+   > Note: The decision of when to remove a Node is a trade-off between optimizing for utilization or the availability of resources. Removing underutilized Nodes improves cluster utilization, but new workloads might have to wait for resources to be provisioned again before they can run.
+
+   > Note: The `bin-packing` profile can be configured only for Shoot cluster with Kubernetes version `>= 1.20`.
+
+   > Note: The `bin-packing` profile is considered as alpha feature. Use it only for evaluation purposes.
+
+   **How it works?**
+   
+   The kube-scheduler is configured with the following bin packing profile:
+
+   ```yaml
+   apiVersion: kubescheduler.config.k8s.io/v1beta3
+   kind: KubeSchedulerConfiguration
+   profiles:
+   - schedulerName: bin-packing-scheduler
+     pluginConfig:
+     - name: NodeResourcesFit
+       args:
+         scoringStrategy:
+           type: MostAllocated
+     plugins:
+       score:
+         disabled:
+         - name: NodeResourcesBalancedAllocation
+   ```
+
+   To impose the new profile a `MutatingWebhookConfiguration` is deployed in the Shoot cluster. The `MutatingWebhookConfiguration` intercepts `CREATE` operations for Pods and sets the scheduler name in the Pod spec to `bin-packing-scheduler`. Then, the Pod gets scheduled by the `bin-packing-scheduler` accordingly. Pods that specify a custom scheduler are not affected.
+
+## Configuring the scheduling profile
+
+The scheduling profile can be configured via the `.spec.kubernetes.kubeScheduler.profile` field in the Shoot:
+
+```yaml
+spec:
+  # ...
+  kubernetes:
+    kubeScheduler:
+      profile: "default" # or "bin-packing"
+```
+
+## Custom scheduling profiles
+
+The kube-scheduler's component configs allows configuring custom scheduling profiles to match the cluster needs. As of today, Gardener supports only two predefined scheduling profiles. The profile configuration in the component config is quite expressive and it is not possible to easily define profiles that would match the needs of every cluster. Because of these reasons there are no plans to add support for new predefined scheduling profiles. If a cluster owner wants to use a custom scheduling profile, then he/she has to deploy (and maintain) own kube-scheduler deployment in the cluster itself.
```

Comment:
```suggestion
The kube-scheduler's component configs allows configuring custom scheduling profiles to match the cluster needs. As of today, Gardener supports only two predefined scheduling profiles. The profile configuration in the component config is quite expressive and it is not possible to easily define profiles that would match the needs of every cluster. Because of these reasons, there are no plans to add support for new predefined scheduling profiles. If a cluster owner wants to use a custom scheduling profile then they have to deploy (and maintain) a dedicated kube-scheduler deployment in the cluster itself.
```